### PR TITLE
chore(macros/GamesSidebar) init zh-CN translation of GamesSidebar

### PR DESCRIPTION
**Pull request by jasonren0403**
## Summary

part of https://github.com/mdn/translated-content/issues/13881

@yin1999 

Noticed that `Efficient_animation_for_web_games` redirects back to [Techniques for game development](https://developer.mozilla.org/en-US/docs/Games/Techniques), so the item may be removed from sidebar? Or it didn't get linked to correct content?

### CODE REVIEW

File: kumascript/macros/GamesSidebar.ejs [hunk 1]
```diff
@@ -337,6 +337,60 @@ const text = mdn.localStringMap({
...
       "Game_monetization": "Monetização de Jogos"
   },
+  "zh-CN": {
+    "Introduction": "简介",
+      "Anatomy": "剖析",
+    "APIs_for_game_development": "游戏开发用到的 API",
+      "Canvas": "Canvas",
+      "CSS": "CSS",
+      "Full_Screen": "全屏",
+      "Gamepad": "Gamepad",
```
Comment (yin1999) [inline kumascript/macros/GamesSidebar.ejs:347]: Should we translate this "游戏控制器"?

File: kumascript/macros/GamesSidebar.ejs [hunk 2]
```diff
@@ -337,6 +337,60 @@ const text = mdn.localStringMap({
...
+      "IndexedDB": "IndexedDB",
+      "JavaScript": "JavaScript",
+      "Pointer_Lock": "Pointer Lock",
```
Comment (yin1999) [inline kumascript/macros/GamesSidebar.ejs:350]: 指针锁定

File: kumascript/macros/GamesSidebar.ejs [hunk 3]
```diff
@@ -337,6 +337,60 @@ const text = mdn.localStringMap({
...
+      "SVG": "SVG",
+      "Typed_Arrays": "类型数组",
```
Comment (yin1999) [inline kumascript/macros/GamesSidebar.ejs:352]: 类型化数组

File: kumascript/macros/GamesSidebar.ejs [hunk 4]
```diff
@@ -337,6 +337,60 @@ const text = mdn.localStringMap({
...
+      "Web_Audio": "Web 音频",
+      "WebGL": "WebGL",
+      "WebRTC": "WebRTC",
+      "Web_Sockets": "WebSockets",
```
Comment (yin1999) [inline kumascript/macros/GamesSidebar.ejs:356]:
```suggestion
      "Web_Sockets": "WebSocket",
```

File: kumascript/macros/GamesSidebar.ejs [hunk 5]
```diff
@@ -337,6 +337,60 @@ const text = mdn.localStringMap({
...
+      "WebVR": "WebVR",
+      "Web_Workers": "Web Worker",
+      "XMLHttpRequest": "XMLHttpRequest",
```
Comment (yin1999) [inline kumascript/macros/GamesSidebar.ejs:359]: Maybe we could replace this (in another PR)?

ref: mdn/content#31010

**Review by yin1999 — COMMENTED** [review]
Some comments for the translation and a note for "De-XHR":

File: kumascript/macros/GamesSidebar.ejs (hunk 2 above)
Comment (jasonren0403) [reply kumascript/macros/GamesSidebar.ejs:350]:
```suggestion
      "Pointer_Lock": "指针锁定",
```

File: kumascript/macros/GamesSidebar.ejs (hunk 1 above)
Comment (jasonren0403) [reply kumascript/macros/GamesSidebar.ejs:347]:
```suggestion
      "Gamepad": "游戏控制器",
```

**Comment by caugner** [comment]
@yin1999 Can you please give the translations a final review and approve if all looks good? 🙏

**Review by yin1999 — APPROVED** [review]
LGTM. Thanks for the note @caugner

**Pull request merged**